internal/provider/utils/connection.go

```diff
@@ -52,11 +52,6 @@ func GetConnection(provider string,
 	connections := make([]models.Connection, 0, len(servers))
 	for _, server := range servers {
 		for _, ip := range server.IPs {
-			if ip.To4() == nil {
-				// do not use IPv6 connections for now
-				continue
-			}
-
```

Comment:
AFAIK the only provider having ipv6 addresses is Mullvad, have you tried and checked it works for both wireguard and openvpn?

Reply:
I did not check openvpn, i'll do it as soon as I can.